### DISCUSSION TITLE
Hardware acceleration when trying both Plex and Jellyfin on the same system.

### DISCUSSION THREAD

**Discussion by mgrimace** [tteck/Proxmox#1531] [Q&A] [answered]
Hi, setting up a new homelab and I'd like to try Jellyfin to see if it's a better fit than Plex, which is what I'm currently using. 

Setup: I have my *arrs, qbittorrent, etc., all on their own Docker LXC and the tteck plex script installed Plex bare-metal in its own, separate LXC with hardware support for my iGPU (intel  i7-8700T). At least I think there's hardware support, Plex requires you to pay to enable it hence why I'm looking into Jellyfin.

Question: can I *also* install the Jellyfin script with hardware support? I don't know if there's a conflict trying to pass the iGPU through to two containers. Can they both run? Do I need to shutdown Plex LXC? Delete it? I'm hesitant to remove Plex in case I want to go back. Thanks in advance for clarifying.

**Comment by MoldyTaint** [comment, marked as answer]
You should be okay running both of them at the same time, I've had two LXC's run at once also. The issue is when you try to do an LXC + VM.

**Reply by mgrimace** [reply]
Thanks so much for the reply MoldyTaint, my issue before was VM + LXC, glad to hear you can do multiple LXCs with acceleration!

**Reply by MoldyTaint** [reply]
I apologies! I didn't' see it say VM earlier. I was pretty tired when I replied. Glad you got it all sorted out though!

**Reply by mgrimace** [reply]
Oh thank you, and no _my apologies,_ I didn't mean to say this question's issue was a VM! The VM thing was an issue I had previously which is why now I was worried about doing and LXC+LXC both with hardware acceleration. Your response cleared up my question here 100%!
